lib/bin/check-file-headers.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const fs = require('fs');
+
+const header = `// Copyright 2017 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+`;
+
+// This might not be the full list in local development: .gitignore includes
+// other files. However, this list should be sufficient for CircleCI.
+const skip = new Set([
+  '.circleci',
+  '.git',
+  '.github',
+  'config',
+  'docs',
+  'lib/bin/.eslintrc.js',
+  'lib/util/quarantine/pkcs7.js',
+  'node_modules',
+  'test',
+  '.eslintrc.json',
+  '.gitignore',
+  '.npmrc',
+  'CONTRIBUTING.md',
+  'LICENSE',
+  'Makefile',
+  'NOTICE',
+  'README.md',
+  'package-lock.json',
+  'package.json'
```

Comment:
The last time I checked, it wasn't possible to add comments in package.json.

Reply:
I wouldn't expect any JSON file to allow comments.  In the current form, the header is probably only applicable to `.js` files

Reply:
Comments in a JSON file would definitely be nonstandard (though I feel like I've seen it somewhere? 🤔). And agreed about the current header only really working for .js files. I'll probably add a similar script to Frontend at some point. But Frontend has .js and .vue files with slightly different headers, so the script will have to account for that somehow.